### DISCUSSION TITLE
I m not bale to fetch the data

### DISCUSSION THREAD

**Discussion by SravyaSuravarjula** [learndevopsonline/learndevopsonline#1577] [Q&A] [answered]
backend public ip 3.89.135.36

frontend public ip  34.201.163.10

mysql public ip 54.237.14.71

**Comment by sreedharm07** [comment]
Hi @SravyaSuravarjula 
can you copy file with backend.service file instead of backend1.service also repo and expence config file names

**Reply by SravyaSuravarjula** [reply, marked as answer]
Its working. i have changed the names

thank you
